src/main/java/edu/hawaii/its/groupings/configuration/SecurityConfig.java

```diff
@@ -184,6 +182,7 @@ public SecurityFilterChain filterChain(HttpSecurity http, CasAuthenticationFilte
 
         http.authorizeHttpRequests((auths) -> auths
                 .requestMatchers(antMatcher("/")).permitAll()
+                .requestMatchers(antMatcher("/announcements/active")).permitAll()
```

Comment:
We can go with setting the `antMatcher` to "/annoucements/** so that it opens more possibilities in the future to filter the announcements by other statuses.

Reply:
The /active is to indicate the status of the messages. In a previous discussion, it was decided that we want to create other endpoints with "/future" or "/expired."

Reply:
Perfect, could you set it to "/accouncements/**" now for the convenience of the person who will implement the other endpoints.